examples/operator/templates/deployment.yaml

```diff
@@ -98,6 +85,26 @@ spec:
           subPath: controller_manager_config.yaml
         - mountPath: /my.ca
           name: secret-volume
+        {{- if .Values.controllerManager.manager.livenessProbe }}
+        livenessProbe: {{- include "tplvalues.render" (dict "value" .Values.controllerManager.manager.livenessProbe "context" $) | nindent 10 }}
+         {{- else }}
+        livenessProbe:
+         httpGet:
+           path: /healthz
+           port: 8081
+         initialDelaySeconds: 15
+         periodSeconds: 20
+        {{- end }}
+        {{- if .Values.controllerManager.manager.readinessProbe }}
```

Comment:
should we template probes only if `probes=true` and leave it as it is by default?

Reply:
ok will do :)

Reply:
@attor I am a bit confused, so if probes= true we template and add values otherwise I would expect to have just the probe in the deployment, correct?

Reply:
we template probes only if probes=true and probes are presented in the source manifest. if any of these conditions is false then we do nothing.